rules/windows/process_creation/proc_creation_win_susp_copy_lateral_movement.yml

```diff
@@ -20,15 +21,23 @@ logsource:
     category: process_creation
     product: windows
 detection:
-    selection1:
+    selection_other_tools:
         Image|endswith:
             - '\robocopy.exe'
             - '\xcopy.exe'
-    selection2:
-        Image|endswith: '\cmd.exe'
+    selection_cmd_img:
+        - Image|endswith: '\cmd.exe'
+        - OriginalFileName: 'Cmd.Exe'
+    selection_cmd_cli:
         CommandLine|contains: 'copy'
-    selection3:
-        Image|contains: '\powershell'
+    selection_pwsh_img:
+        - Image|contains:
+            - '\powershell.exe'
+            - '\pwsh.exe'
+        - OriginalFileName:
+            - 'PowerShell.EXE'
+            - 'pwsh.dll'
+    selection_pwsh_cli:
         CommandLine|contains:
             - 'copy-item'
             - 'copy'
```

Comment:
I cannot make github to edit "unshown" areas. Please add:
```
'move-item'
' mi '

```

Reply:
Done